internal/osbuildconfig/osbuildconfig_handler.go

```diff
@@ -7,30 +7,35 @@ package osbuildconfig
 
 import (
 	"fmt"
+	osbuildv1alpha1 "github.com/project-flotta/osbuild-operator/api/v1alpha1"
```

Comment:
Please move the imports to their corresponding location

Reply:
Done